### PR TITLE
Reduce image size

### PR DESCRIPTION
**Pull request by tyuio9**

### CODE REVIEW

**Comment by tyuio9** [comment]
before:
```
docker images explorer-kit
REPOSITORY     TAG       IMAGE ID       CREATED          SIZE
explorer-kit   latest    e192231e2cc1   23 seconds ago   2.01GB
```

**Comment by tyuio9** [comment]
after:
```
docker images explorer-kit
REPOSITORY     TAG       IMAGE ID       CREATED          SIZE
explorer-kit   latest    669d2e7c2d34   30 seconds ago   507MB
```

**Comment by tyuio9** [comment]
My primary intention is to reduce the time to download the data when you pull the image often enough on a clean machine, e.g., GitHub Actions runners.

File: Dockerfile [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
 COPY . .
 
-# Install workspace dependencies
-RUN pnpm install
+RUN apk add --update --upgrade --no-cache \
+        alpine-sdk \
+        python3 \
+        ca-certificates \
+    && \
+    npm install -g pnpm@v8.6.10 && \
```
Comment (denisglotov) [inline Dockerfile:15]: just a Q: why pin the version? wouldn't we forget to update pnpm version here?

**Review by denisglotov — APPROVED** [review]
👍

File: Dockerfile (hunk 1 above)
Comment (tyuio9) [reply Dockerfile:15]: Good call. If you use a different one, you will see a warning. We specify the `packageManager` version in the `package.json` file.

I'm happy to bump it and use the latest NodeJS LTS. What do you think?

Comment (denisglotov) [reply Dockerfile:15]: sounds good, please proceed

**Comment by kimpers** [comment]
I'm going to go ahead and merge this as it's already approved

**Pull request merged**